app/services/food_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodService
+  def self.call_for_food(food)
+    response = conn.get("/fdc/v1/foods/search?api_key=#{ENV['PARKS_API_KEY']}&query=#{food}&Size=10&brandOwner")
+    parse_json(response)
+  end
+
+private
+
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov") do |faraday|
+      faraday.headers["X-API-KEY"] = ENV['PARKS_API_KEY']
```

Comment:
so in here, you can just change this to `faraday.param["api_key"] = ENV['PARKS_API_KEY']` (but maybe change the env variable name to match the api key it's storing.